### ISSUE TITLE
when export catboost model to python file get a wrong `float_feature_count`

### ISSUE THREAD

**Issue by coboe** [catboost/catboost#461]
Problem: {problem description}
catboost version: {0.9.1.1}
Operating System: {Win 10}
CPU: {i3-7100}

At first, my train data set has 207 numerical features, include 12 categorical feature.
after train the `CatBoostClassifier` model, export to python bu `save_model('./catboost.py', format="python")` ,
then, the numerical features count become 152 at `catboost_model` class, and categorical feature count is correct.

is any numerical features is not used?
how can I find which numerical features is not used?

**Comment by coboe** [comment]
Thank you!

Sometimes, it work fine.
I train catboost model on Titantic data, the exported python code is correct.

**Comment by annaveronika** [comment]
Yes, the bug appears if some feature is not used in the model, so it works fine if all the features are used.
